#
time stamp
uint32 waypoint_seq
float64 waypoint_x
float64 waypoint_y
float64 own_x
float64 own_y
float64 own_yaw

float64 tf_waypoint_x
float64 tf_waypoint_y
float64 tf_own_x
float64 tf_own_y
float64 cross_track_error

float64 Kp
float64 Ki
float64 Kd
float64 look_ahead_dist

float64 p
float64 i
float64 d
float64 steering_ang #[degree]
float64 translation
float64 steering
